curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-recursion-by-building-a-decimal-to-binary-converter/6464ab8c06ea92f30bc548d5.md

```diff
@@ -10,7 +10,7 @@ dashedName: step-102
 For the last animation object, add the property `msg` set to the following string:
 
 ```md
-decimalToBinary(5) returns "10" + 1 (5 % 2). Then it pops off the stack.
+`decimalToBinary(5) returns "10" + 1 (5 % 2). Then it pops off the stack.`
```

Comment:
```suggestion
`"decimalToBinary(5) returns '10' + 1 (5 % 2). Then it pops off the stack."`
```

Reply:
There are triple backticks already, I've removed the single backticks around double quotes